src/testshade/optixgridrender.cpp

```diff
@@ -684,16 +684,16 @@ OptixGridRenderer::good(TextureHandle* handle OSL_MAYBE_UNUSED)
 /// Given the name of a texture, return an opaque handle that can be
 /// used with texture calls to avoid the name lookups.
 RendererServices::TextureHandle*
-OptixGridRenderer::get_texture_handle(ustring filename OSL_MAYBE_UNUSED,
-                                      ShadingContext* shading_context
-                                          OSL_MAYBE_UNUSED)
```

Comment:
Is it ok, that the OSL_MAYBE_UNUSED was removed?

Reply:
filename is fine because it *is* used.

shading_context is fine because I commented out the parameter name.